src/lib.rs

```diff
@@ -384,22 +519,18 @@ const SEP: char = '1';
 
 /// Encoding character set. Maps data value -> char
 const CHARSET: [char; 32] = [
-    'q','p','z','r','y','9','x','8',
-    'g','f','2','t','v','d','w','0',
-    's','3','j','n','5','4','k','h',
-    'c','e','6','m','u','a','7','l'
+    'q', 'p', 'z', 'r', 'y', '9', 'x', '8', 'g', 'f', '2', 't', 'v', 'd', 'w', '0', 's', '3', 'j',
```

Comment:
It looks like these have been auto-formatted, which makes them a little harder to visually verify against the BIP-0173 spec and other implementations. Not a deal-breaker, since errors will show up in test, but if there's a clean way to skip auto-formatting / linting for these few lines (and for `CHARSET_REV` below), I would prefer that.

Reply:
I reverted the changes and added an ignore attribute.

Reply:
Looks like tests are failing from that new attribute.

Reply:
Yeah, I'm already searching for the right attribute to deactivate that, I used it recently but can't remember, I guess it's `#[allow(unknown_attributes)]`.

Reply:
I ended up just removing the attributes. `unknown_attributes` requires a feature only available on nightly.